<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartialGameWindow</class>
 <widget class="QDialog" name="PartialGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pin2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>370</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin9">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>60</y>
     <width>53</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin8">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>170</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin5">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>260</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>211</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Súčet  kolkov</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin6">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>260</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin1">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>460</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pin7">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>170</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>191</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Počet hodov</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="roundsLCD">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>200</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="scoreLCD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>200</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pin3">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>370</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ninepins.qrc">
     <normaloff>:/images/kolok1.png</normaloff>
     <normalon>:/images/kolok2.png</normalon>:/images/kolok1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>173</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pointsLCD">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>600</y>
     <width>200</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>70</y>
     <width>131</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Dorážka</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stavButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>500</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Upravit stav</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stavanieButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>500</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stavanie</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="koniecButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>500</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Koniec Hry </string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ninepins.qrc"/>
 </resources>
 <connections/>
</ui>
